crates/sui-types/src/dynamic_field.rs

```diff
@@ -298,47 +296,43 @@ where
     Ok(id)
 }
 
-fn get_dynamic_field_object_from_store_impl<K>(
-    object_store: &dyn ObjectStore,
-    parent_id: ObjectID,
-    key: &K,
-    key_type_tag: &TypeTag,
-) -> Result<Object, SuiError>
+pub fn deserialize_dynamic_field<K, V>(object: Object) -> Result<Field<K, V>, SuiError>
```

Comment:
Is there a reason to still keep the `deserialize_dynamic_field` functions

Reply:
inlined them